testdata/scripts/runtime-related.txt

```diff
@@ -0,0 +1,57 @@
+# test that we used all necessary dependencies
+[!darwin] exec bash runtime-related-tested.sh
+
+env GOPRIVATE=*
+garble -literals build
+
+-- runtime-related.sh --
```

Comment:
should we delete scripts/runtime-related.sh then?

Reply:
Probably